solr/test-framework/src/java/org/apache/solr/util/BadHdfsThreadsFilter.java

```diff
@@ -41,6 +41,10 @@ public boolean reject(Thread t) {
       return true;
     } else if (name.startsWith("nioEventLoopGroup")) {
       return true; // netty threads waiting for a web server stop confirm that won't happen
+    } else if (name.startsWith("globalEventExecutor")) { // HADOOP-11219 (Hadoop and Netty 4)
```

Comment:
This... shouldn't leak? Why does it leak? According to https://netty.io/4.0/api/io/netty/util/concurrent/GlobalEventExecutor.html it will clean itself up after a second of inactivity.

Reply:
I think the issue is `ThreadLeakLingering` was adjusted to be really short (less than 1 second) for a bunch of the Hadoop tests and sometimes we run into `globalEventExecutor` not having been shut down yet. 

See 1c80803a

Reply:
FWIW globalEventExecutor is leaking in a few HDFS tests - without this change being merged in yet. I haven't looked into it just saw the jenkins emails go by.

Reply:
I bumped up the thread leak linger to minimum 1 second to allow this to shut down. I haven't seen this failure since.